### DISCUSSION TITLE
Number of processes

### DISCUSSION THREAD

**Discussion by gerep** [kovidgoyal/kitty#5275] [Q&A] [answered]
I have Kitty started just to run `htop` and I see it has a few processes running.

Is that expected? If so, why?

![image](https://user-images.githubusercontent.com/181501/178813701-70549969-6032-4991-8ad9-b7f0a4601b3f.png)

**Comment by page-down** [comment, marked as answer]
That looks like multiple threads.
Press F2 and check "Display options" -> "Hide userland process threads".
The latest version of kitty will have two processes after it runs, the extra one is the prewarm process, which is for quickly starting kittens.
